src/auth.ts

```diff
@@ -1,18 +1,46 @@
 import { PrismaAdapter } from "@auth/prisma-adapter";
-import NextAuth from "next-auth";
+import { Role } from "@prisma/client";
+import NextAuth, { DefaultSession } from "next-auth";
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { JWT } from "next-auth/jwt";
```

Comment:
```suggestion
import { type JWT } from "next-auth/jwt";
```
now it wont complain

Reply:
sadly it still complains :/

Reply:
damn thats a really weird behavior then xD its not even a var but a type